### PR TITLE
Fix DnsCache.MachineName on Docker Linux container

### PR DESCRIPTION
**Pull request by mconnew**
Remove dead code from DnsCache
Add another stage to pipeline to sign symbol packages

### CODE REVIEW

File: templates/CodeSignStage.yml [hunk 1]
```diff
@@ -42,6 +35,19 @@ stages:
               --descriptionUrl "https://github.com/CoreWCF/CoreWCF"
             displayName: Sign packages
 
+          - pwsh: |
```
Comment (AAATechGuy) [inline templates/CodeSignStage.yml:38]: don't have much expertise on this change.

File: src/Common/src/CoreWCF/Channels/DnsCache.cs [hunk 2]
```diff
@@ -1,26 +1,20 @@
...
+    internal static class DnsCache
     {
-        const int mruWatermark = 64;
-        static MruCache<string, DnsCacheEntry> resolveCache = new MruCache<string, DnsCacheEntry>(mruWatermark);
-        static readonly TimeSpan cacheTimeout = TimeSpan.FromSeconds(2);
+        private const string LinuxProcHostnamePath = "/proc/sys/kernel/hostname";
 
         // Double-checked locking pattern requires volatile for read/write synchronization
-        static volatile string machineName;
+        private static volatile string machineName;
```
Comment (AAATechGuy) [inline src/Common/src/CoreWCF/Channels/DnsCache.cs:15]: not sure, if this was needed in C#. 
https://stackoverflow.com/questions/1964731/the-need-for-volatile-modifier-in-double-checked-locking-in-net

**Review by AAATechGuy — APPROVED** [review]
signing off with comments.

File: src/Common/src/CoreWCF/Channels/DnsCache.cs (hunk 2 above)
Comment (AAATechGuy) [reply src/Common/src/CoreWCF/Channels/DnsCache.cs:15]: nvm, the comment was specific to x86/x64 processor architecture

Comment (mconnew) [reply src/Common/src/CoreWCF/Channels/DnsCache.cs:15]: Yep, the JIT is smart enough to no-op it on x86 and x64 platforms and to do a volatile read on platforms which need it. I think ARM has a weaker memory model.

**Pull request merged**